main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
 ```
 
-## `E(zoe).install(bundle)`
-- `bundle` `{SourceBundle}`
-- Returns: `{Promise<Installation>}`
+## E(Zoe).install(bundle)
+- **bundle** **SourceBundle**
+- Returns: **Promise&lt;Installation>**
 
 Takes bundled source code for a Zoe contract as an argument and installs the code on Zoe.
-Returns an `installation` object.
-
-An `installation` is an object with one property:
-- `bundle`:  The contract source code, accessible via `bundle.source`, and other info.
+Returns a **Promise** for an **Installation** object.
 
 ```js
 // bundleSource takes source code files and
 // bundles them together in the format install expects.
-import bundleSource from '@endo/bundle-source';
+import bundleSource from '@endto/bundle-source';
 const bundle = await bundleSource(pathResolve(`./src/contract.js`));
-const installationP = await E(zoe).install(bundle);
+const installationP = await E(Zoe).install(bundle);
 ```
 
-## `E(zoe).getInstance(invitation)`
-- `invitation` `{Invitation}`
-- Returns: `{Promise<Instance>}`
+## E(Zoe).getConfiguration()
+- Returns: **Promise&lt;Object>**
+
+Returns the configuration settings for the Zoe contract.
+
+## E(Zoe).getFeeIssuer()
+- Returns: **Promise&lt;[Issuer](/reference/ertp-api/issuer.md)>**
+
+Returns a **Promise** for an **Issuer** that can mint ISTs.
+
+## E(Zoe).installBundleID(bundleId)
+- bundleId **BundleId**
+- Returns: **Promise&lt;Installation>**
 
-Returns a `Promise` for the contract `instance` the `invitation` is part of.
+Reserved for future use.
 
-While `instances` are opaque objects, you can get information about them via
+
+## E(Zoe).getOfferFilter(instance)
+- **instance** **[Instance](./zoe-data-types.md#instance)**
+- Returns: **Array&lt;String>**
+
+Returns all the offer keywords that have been disabled, if any. Offer keywords may be disabled if they prove problematic in some fashion, or to debug undesired behavior.
+
+## E(Zoe).getBundleIDFromInstallation(installation) 
+- **installation** **Installation**
+- Returns: **Promise&lt;BundleId>**
+
+Reserved for future use.
+
+## E(Zoe).getInstance(invitation)
+- **invitation** **Invitation**
+- Returns: **Promise&lt;[Instance](./zoe-data-types.md#instance)>**
+
+Returns a **Promise** for the contract **instance** the **invitation** is part of.
+
+While **Instances** are opaque objects, you can get information about them via
 these methods:
-- `getBrands()`
-- `getTerms()`
-- `getIssuers()`
-- `getPublicFacet()`
+
+- **getBrands()**
+- **getTerms()**
+- **getIssuers()**
+- **getPublicFacet()**
 
 ```js
-const instance = await E(zoe).getInstance(invitation);
+const instance = await E(Zoe).getInstance(invitation);
 ```
 
-## `E(zoe).getInstallation(invitation)`
-- `invitation` `{Invitation}`
-- Returns: `{Promise<Installation>}`
+## E(Zoe).getProposalShapeForInvitation(invitation) 
+- **invitation** **Invitation**
+- Returns: **Promise&lt;Pattern>**
+
+Returns a **Promise** for the **Pattern** that the **Invitation's** **Proposal** adheres to.
```

Comment:
`Pattern` is an undefined type.

Reply:
Yes it is, but that's intentional. (For now.) Describing what the Pattern object truly represents is a major work item, so I'm just "hand-waving" the meaning of "Patterns" for now.